dist/images/masonry-modal/arrow-left.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Is it possible to remove this?

Reply:
Yea, this SVG needs to be cleaned.

Reply:
This is what I've used to clean up SVGs: https://jakearchibald.github.io/svgomg/